README.md

```diff
@@ -16,6 +16,7 @@ Game networking is a subset of computer networking that specifically covers data
 
 ## Articles
 
+- [The DOOM III Network Architecture](http://mrelusive.com/publications/papers/The-DOOM-III-Network-Architecture.pdf) - Architecture improves upon previous network architectures used in the Quake III Arena
```

Comment:
Bad ordering!
Should insert between "The Case of the Quake Cheats" and "Unreal Engine 4 Network Compendium".

> - Entry name in title-case and list in alphabetical order.

Reply:
Bad description format.
> - Entry description starts with uppercase and ends with a period.

Reply:
Wraps to next-line.
> - Entry does not wrap to next-line.

Reply:
Entry description:
- starts with uppercase.
- ends with a period.
Entry name:
- in title-case.
- list in alphabetical order.

So you may point to specific rules instead of `and`

>> Wraps to next-line.
Do not get it.